### PR TITLE
Rename the maxMinterms configuration

### PR DESCRIPTION
**Pull request by scottinet**
# Description

The `maxMinterms` configuration and its explanation are misleading: this limit is applied to filter's conditions, not to its minterms.
The goal of that configuration is to limit the complexity of the truth tables to build to compute the filter's DNF.

New name: `maxConditions`

### CODE REVIEW

**Review by Shiranuit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1040,7 +1040,7 @@ constructor(options: KoncordeOptions = null)
 
 | Name | Type | Default |Description                      |
 |------|------|---------|---------------------------------|
-|`maxMinTerms`| `Number` | `256` | The maximum number of conditions a filter can hold after being canonicalized in its [CDNF](https://en.wikipedia.org/wiki/Canonical_normal_form) form. It is advised to test performances and memory consumption impacts before increasing this value. If set to 0, no limit is applied. |
+|`maxConditions`| `Number` | `50` | The maximum number of conditions a filter can hold. It is advised to test performances and memory consumption impacts before increasing this value. If set to 0, no limit is applied. |
```
Comment (Aschen) [inline README.md:1043]: You should add a note saying that the limit is applied after normalization

**Review by Aschen — APPROVED** [review]

**Review by alexandrebouthinon — COMMENTED** [review]
Missing failure scenario for unit tests?

File: README.md (hunk 1 above)
Comment (scottinet) [reply README.md:1037]: The wiki mentions that. As discussed a few minutes ago, I'll submit another PR removing most of the README's content and redirect to the wiki.

**Comment by scottinet** [comment]
@alexandrebouthinon > you're right. Tests added.

**Review by alexandrebouthinon — APPROVED** [review]

**Pull request merged**